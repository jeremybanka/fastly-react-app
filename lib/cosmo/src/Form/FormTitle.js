/* @flow */

import * as React from "react";
import { Box, Text } from "../";

type Props = { children: React.Node };

const FormTitle = (props: Props) => (
  <Box>
    {/* hmm */}
    <Text as="h4">{props.children}</Text>
  </Box>
);

export default FormTitle;
