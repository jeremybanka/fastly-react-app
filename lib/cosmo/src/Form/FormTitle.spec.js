/* @flow */

import React from "react";
import { render } from "enzyme";
import FormTitle from "./FormTitle";

describe("FormTitle test suite", () => {
  it("renders the stored snapshot", () => {
    const component = <FormTitle>My Title</FormTitle>;
    expect(render(component)).toMatchSnapshot();
  });
});
