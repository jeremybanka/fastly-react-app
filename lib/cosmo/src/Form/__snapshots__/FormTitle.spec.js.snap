// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormTitle test suite renders the stored snapshot 1`] = `
<div
  class="Box-sc-1v2etov-0 kAhoak"
>
  <h4
    class="Text-sc-6c1xd4-0 eVgFRd"
  >
    My Title
  </h4>
</div>
`;
